test/fixtures.rb

```diff
@@ -339,7 +339,7 @@ class StoreLayout
       include Store::Helpers::AssetTagHelpers
 
       def head
-        %(<meta name="lotusrb-version" content="0.3.1">)
+        Lotus::Utils::Escape::SafeString.new %(<meta name="lotusrb-version" content="0.3.1">)
```

Comment:
notice lotus version 0.3.1 in this tag, do you think this tag is useful or secure? I mean, you're telling the world what lotus version you are using.